### DISCUSSION TITLE
I want to use a pre-trained model to train on my data or use my data + voxceleb1 data set to train

### DISCUSSION THREAD

**Discussion by kevin00616** [speechbrain/speechbrain#2229] [Q&A]
As title,How can I do it?
Can someboday help me do it?
I need to know more detailed steps.
Thanks!!

**Comment by TParcollet** [comment]
Hi, please describe exactly what you would like to do. But first, have a look at the list of SB tutorials on the website. Wether it is Basics or Advanced we have some on pretraining.

**Comment by kevin00616** [comment]
Hello @TParcollet 
My training is to train the ECAPA-TDNN model.

I have two questions want to ask.

1.How can I add my own data to the voxceleb1 data set for training?
2.How can I use the pre-trained model of ECAPA-TDNN to train on my own data set? (meaning:Fine Tuning)
